godot_project/editor/controls/general/nano_file_dialog.gd

```diff
@@ -74,21 +74,21 @@ func _sort_tree_items_alphabetically(in_item_a: TreeItem, in_item_b: TreeItem) -
 
 func _get_file_dialog_line_edit() -> LineEdit:
 	var body_container: VBoxContainer = get_child(3, true) as VBoxContainer
-	assert(is_instance_valid(body_container), "Invalid child of FileDialog, Did engine version change from 4.1.1?")
-	var footer_container: HBoxContainer = body_container.get_child(3, true) as HBoxContainer
-	assert(is_instance_valid(footer_container), "Invalid child of body_container, Did engine version change from 4.1.1?")
+	assert(is_instance_valid(body_container), "Invalid child of FileDialog, Did engine version change from 4.4.1?")
+	var footer_container: HBoxContainer = body_container.get_child(4, true) as HBoxContainer
```

Comment:
This line is the actual fix, the rest is only updating Godot version in the asser message